### PR TITLE
CDD-2222: WHA cms component

### PR DESCRIPTION
**Pull request by phill-stanley**
# Description

This PR includes the a weather health alert card card in the cms. This card includes a `title`, `sub-title` and a `choiceblock` to select the current alert type `Heat` or `Cold`.

Fixes #CDD-2222

---

## Type of change

Please select the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Tech debt item (this is focused solely on addressing any relevant technical debt)

---

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests at the right levels to prove my change is effective
- [x] I have added screenshots or screen grabs where appropriate
- [ ] I have added docstrings in the correct style [(google)](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings)

### CODE REVIEW

**Comment by phill-stanley** [comment]
Screenshot 2024-09-20 at 15 01 13

File: cms/dynamic_content/cards.py [hunk 1]
```diff
@@ -40,6 +31,37 @@ class Meta:
...
+class WHAlerts(models.TextChoices):
+    HEAT = "Heat"
+    COLD = "Cold"
+
+    @classmethod
+    def get_alerts(cls) -> tuple[tuple[str, str]]:
+        return tuple((alert.value, alert.value) for alert in cls)
+
+
+class WHACard(blocks.StructBlock):
```
Comment (A-Ashiq) [inline cms/dynamic_content/cards.py:52]: Nitpick: How do you feel about using the full name and not the abbreviation?
```
WeatherHealthAlertsCard
```
It doesn't feel too verbose and it cuts out the cognitive load of an abbreviation

File: cms/dynamic_content/help_texts.py [hunk 2]
```diff
@@ -159,6 +159,10 @@
 The title to display for this component. 
 Note that this will be shown in the hex colour #505A5F
 """
+SUB_TITLE_FIELD: str = """
+The Sub title to display for this component.
```
Comment (A-Ashiq) [inline cms/dynamic_content/help_texts.py:163]:
```suggestion
The sub title to display for this component.
```

File: cms/dynamic_content/cards.py (hunk 1 above)
Comment (phill-stanley) [reply cms/dynamic_content/cards.py]: Yeah good point, I'm use to seeing the abbreviation but probably best to give the full name here. I've updated this and the display name for the `ContentCards` section.

**Review by A-Ashiq — APPROVED** [review]

**Review by zesh92 — APPROVED** [review]
LGTM

**Pull request merged**